app/services/order_row_importer.rb

```diff
@@ -295,7 +296,17 @@ def validate_existing_order
     end
   end
 
-  # Projects adds a custom attribute and overrides this method to validate it
   def validate_custom_attributes
+    if field(:project_name).present? && project.nil?
```

Comment:
I think this should be in its own method like the others, maybe `validate_project_name`